modules/tensor_mechanics/include/materials/ADRadialReturnStressUpdate.h

```diff
@@ -124,6 +124,7 @@ class ADRadialReturnStressUpdate : public ADStressUpdateBase<compute_stage>,
   ADMaterialProperty(Real) & _effective_inelastic_strain;
   const MaterialProperty<Real> & _effective_inelastic_strain_old;
   Real _max_inelastic_increment;
+  const bool _apply_strain;
```

Comment:
..not thrilled about this option because it seems like it could be accidentally misused

Reply:
I can move this to advanced input parameter group, or even hide it.